.github/workflows/release.yml

```diff
@@ -37,16 +37,12 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Checkout copa-action repository
-        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
-        with:
-          repository: project-copacetic/copa-action
-          ref: main
-
-      - name: Set up Docker
-        uses: docker/setup-buildx-action@4c0219f9ac95b02789c1075625400b2acbff50b1 # v2.9.1
-
-      - name: Build and push copa-action image with new version
+      - name: Trigger Repo Dispatch for Copa Action Release
         run: |
-          tag="$(echo "${{ github.ref }}" | tr -d 'refs/tags/v')"
-          docker buildx build --build-arg copa_version=${tag} -t ghcr.io/project-copacetic/copa-action:v"$tag" --push .
+            tag="$(echo "${{ github.ref }}" | tr -d 'refs/tags/v')"
+            payload="{\"event_type\": \"copa_release\", \"client_payload\": { \"version\": \"$tag\" }}"
+            curl -H "Accept: application/vnd.github.everest-preview+json" \
+            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
```

Comment:
how does this token work to trigger copa-action?

Reply:
I tested it with this token on these runs: https://github.com/ashnamehrotra/copacetic/actions/runs/5969797230/job/16196305199 & https://github.com/ashnamehrotra/copa-action/actions/runs/5969828753/job/16196388231

But this was set as a personal access token so it gave access to both repos. Is there a way to scope it to the project-copacetic org? I'm not sure if I have access to see the tokens right now

Reply:
right, you can do that with a PAT but not with repo specific token. I am not sure if you can access cross-repos with a token from a repo

Reply:
I wasn't able to figure out a good way to do this without access issues. For now I updated https://github.com/project-copacetic/copa-action/pull/14 to release manually. Another way to do this automatically could be to use [Github Apps](https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/making-authenticated-api-requests-with-a-github-app-in-a-github-actions-workflow), but I am not sure if it is worth creating an app for the authentication?